test.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
```

Comment:
Sorry. I didn't find it in the script. Now I see it but it's hard to recognise it.